Bugsnag/Storage/BSGFileLocations.h

```diff
@@ -49,9 +53,43 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @property (readonly, nonatomic) NSString *systemState;
 
+/**
+ * Returns `YES` if the receiver uses an atomic subdirectory, `NO` if it uses the shared default directory.
+ */
+@property (readonly, nonatomic) BOOL usesAtomicSubdirectory;
```

Comment:
I worry a bit about the use of "atomic" in these APIs. It feels like the implementation (use of a lock to coordinate multiple processes) is what's being named after, rather than the _purpose_. Feels like `usesSubdirectory` would do the job just as well? Maybe terming things "sub-process" instead of "atomic" would work better?